willa/web/cas_provider.py

```diff
@@ -25,6 +27,31 @@
     raise ValueError(f'Unknown CalNet environment {CALNET_ENV}!')
 
 
+class CASForbiddenException(HTTPException):
+    """Exception class for when a CAS-authenticated user is not authorized."""
+    def __init__(self, status_code: int, detail: Any = None,
+                 headers: Optional[Dict[str, str]] = None) -> None:
+        super().__init__(
+            status_code=status_code, detail=detail, headers=headers
+        )
+
+
+@app.exception_handler(CASForbiddenException)
+async def cas_forbidden_exception_handler(
+    request: Request, # pylint: disable=unused-argument
```

Comment:
Typically, unused arguments are prefixed with an underscore.  That should also silence the warning.

```suggestion
    _request: Request,
```

Reply:
fixed in 67116e7